src/app/ToDo/state/todo.action.ts

```diff
@@ -1,28 +1,38 @@
+import { Update } from '@ngrx/entity';
 import { createAction, props } from '@ngrx/store';
-import ToDo from './todo.model';
+import { ToDo } from './todo.model';
 
-export const GetToDoAction = createAction('[ToDo] - Get ToDo');
+export enum ActionTypes {
+  LoadTodos = '[Todo] Load Todos',
+  LoadTodosSuccess = '[Todo] Load Todos Success',
+  AddTodo = '[Todo] Add Todo',
+  UpdateTodo = '[Todo] Update Todo',
+  DeleteTodo = '[Todo] Delete Todo',
+  Error = '[ToDo] - Error'
```

Comment:
It's more information.

For our actions, we write the name of the action directly in the method.
For example:
```javascript
export const LoadTodos = createAction(
 '[Todo] Load Todos'
);
```

Reply:
Nice!